SpatialGDK/Documentation/content/get-started/gdk-template.md

```diff
@@ -57,16 +57,14 @@ To use the Starter Template, you must build the GDK for Unreal module dependenci
 1. In the Solution Explorer window, right-click on **`<YourProject>`** and select **Build**.
 1. When Visual Studio has finished building your project, right-click on **`<YourProject>`** and select **Set as StartUp Project**.
 1. Press F5 on your keyboard or select **Local Windows Debugger** in the Visual Studio toolbar to open your project in the Unreal Editor.<br/>
-![Visual Studio toolbar]({{assetRoot}}assets/set-up-template/template-vs-toolbar.png)
+![Visual Studio toolbar]({{assetRoot}}assets/set-up-template/template-vs-toolbar.png)<br/>
 _Image: The Visual Studio toolbar_
 
 Note: Ensure that your Visual Studio Solution Configuration is set to **Development Editor**.
 
-
-
 ### Deploy your project 
 
-To test your project, you can launch a [local deployment (SpatialOS documentation)](https://docs.improbable.io/reference/13.5/shared/glossary#local-deployment) or a [cloud deployment (SpatialOS documentation)](https://docs.improbable.io/reference/13.5/shared/glossary#cloud-deployment).  
+To test your project, you can launch a [local deployment (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary#local-deployment) or a [cloud deployment (SpatialOS documentation)](https://docs.improbable.io/reference/latest/shared/glossary#cloud-deployment).  
 
 #### Deploy locally with multiple clients
 
```

Comment:
I can't do this because I can't edit the bottom of the page, but can you add this to the bottom of the page?

<br/>
<br/>
-------------
2019-03-15 Page updated with limited editorial review